Dockerfile

```diff
@@ -0,0 +1,61 @@
+###############################################################################
```

Comment:
Why do we commit this Dockerfile to the repo? Can't we rely on `flink-docker`?

Reply:
It is a little different than the one on the <code>flink-docker</code> repo in the way it installs the Flink.

Reply:
True, I haven't seen the <code>dev-master</code> branch. I am removing the docker stuff and updating the documentation.